packages-tests/BetterPhpDocParser/PhpDocParser/TagValueNodeReprint/FixtureInlineHtml/expected-inline-html.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\BetterPhpDocParser\PhpDocParser\TagValueNodeReprint\FixtureModif;
+
+?>
+<h1><?php echo $hello; ?></h1>
+-----
+<?php
+/**
+ * @var string $hello
+ */
+?>
```

Comment:
It seems everything is removed here and replaced by `@var` type. 
It should be original code + `@var` instead, right?

Reply:
I thought so too, but it seems the phpdoc testcases only render the actual doc being printed not the overall final source (inspired by other tests in the same die)

Reply:
See the actual vs. expected comparisons within the testcase

Reply:
This is very confusing at first sight.
Better use `AbstractRectorTestCase` for this with before/after content instead.

Reply:
but I have no rector to invoke here. the changes of this PR only affect "helper classes" not rector rule itself?

Reply:
ok, I guess I will construct a fixture-rector justs to test the case.